app/assets/stylesheets/apply.scss

```diff
@@ -55,6 +55,26 @@
   background-size:cover;
 }
 
+.gear-img {
+  background: image-url('apply/gears.svg');
+  background-size: cover;
+  height: 779px;
```

Comment:
Properties should be ordered background, background-size, bottom, height, left, position, width, z-index